src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1006,7 +1018,10 @@ class DocBaseViewer extends BaseViewer {
     loadUI() {
         this.controls = new Controls(this.containerEl);
         this.pageControls = new PageControls(this.controls, this.docEl);
+        this.zoomControls = new ZoomControls(this.controls);
         this.pageControls.addListener('pagechange', this.setPage);
+        this.zoomControls.addListener(ZOOM_CONTROLS_EVENTS.zoomin, this.zoomIn);
```

Comment:
Do these need to be event-based or can we pass in `this.zoomIn` as a callback option to the ZoomControls constructor? Is it the former for consistency with PageControls? Do we want that to be the standard?

Reply:
I'll move them to be callback options which should help with the cleanup of the listeners as well